website/project/model.py

```diff
@@ -4634,6 +4636,14 @@ def is_rejected(self):
         else:
             return False
 
+    @property
+    def logs(self):
+        logs = sorted(
+            [(k, v) for k, v in self.status_logs.iteritems()],
+            key=lambda x: x[0]
+        )
+        return ['{} on {}'.format(v.title(), k) for k, v in logs]
```

Comment:
I'm pretty sure that `[(k,v) for k, v in self.status_log.iteritems()]` gives you the same result as `self.status_log.iteritems()`, a list of tuples of k, v.